msteams-platform/bots/how-to/get-teams-context.md

```diff
@@ -73,7 +73,7 @@ async def _show_members(
 
 # [JSON](#tab/json)
 
-You can directly issue a GET request on `/v3/conversations/{conversationId}/pagedmembers?pageSize={pageSize}&continuationToken={continuationToken}`, using the value of `serviceUrl` as the endpoint. The value of `serviceUrl` tends to be stable but can change. When a new message arrives, your bot should verify its stored value for `serviceUrl`.
+You can directly issue a GET request on `/v3/conversations/{conversationId}/pagedmembers?pageSize={pageSize}&continuationToken={continuationToken}`, using the value of `serviceUrl` as the endpoint. The value of `serviceUrl` tends to be stable but can change. When a new message arrives, your bot should verify its stored value for `serviceUrl`. The response payload will also inform is the user is a regular or an anonymous user.
 
 ```http
 GET /v3/conversations/19:ja0cu120i1jod12j@skype.net/pagedmembers?pageSize=100&continuationToken=asdfasdfalkdsjfalksjdf
```

Comment:
`anonymous user` can only exist in a meeting thread. So if we want to show an example payload of Get Member(s) API in anonymous user scenarios, I think the conversation id of a thread should be a meeting thread id, which always starts with "19:meeting_". 

I pinged Leon on this issue to double confirm. If my understanding is correct, we can update the given examples with this sample meeting thread id - 19:meeting_N2QzYTA3YmItYmMwOC00OTJmLThkYzMtZWMzZGU0NGIyZGI0@thread.v2